src/index.js

```diff
@@ -17,8 +17,9 @@ function capitalizeFirstLetter(string) {
 function checkEndpoint(url, endpoints) {
 	const parser = window.document.createElement('a');
 	parser.href = url;
+	const host = parser.host ? parser.host : window.location.host;
```

Comment:
> What about verifying that the url is relative?

To be honest, I'd rather use a library that write my own (guaranteed to fail on some corner case) is relative check.